tests/scripts/pulp_rpm/test_rpm_sync_publish.sh

```diff
@@ -40,6 +40,10 @@ expect_succ curl "$curl_opt" --head --fail "$PULP_BASE_URL/pulp/content/cli_test
 expect_succ pulp rpm repository version list --repository "cli_test_rpm_repository"
 expect_succ pulp rpm repository version repair --repository "cli_test_rpm_repository" --version 1
 
+expect_succ pulp rpm repository update --name "cli_test_rpm_repository" --retain-package-versions 2
+expect_succ pulp rpm repository show --name "cli_test_rpm_repository"
+test "$(echo "$OUTPUT" | jq -r '.retain_package_versions')" = "2"
```

Comment:
To disable that function, do you nullify it, or will it be set to 0, -1, ...?
Can you add a test for that?

Reply:
Set to 0 turns it off - will add a test, good call.

Reply:
In that case (also being an integer) we probably don't need a test. For nullifying there can be interesting side effects.

Reply:
repushed a change constraining retain-package-versions to integer